src/us/kbase/narrativejobservice/sdkjobs/SDKLocalMethodRunner.java

```diff
@@ -254,7 +254,14 @@ public void run() {
             String adminTokenStr = System.getenv("KB_ADMIN_AUTH_TOKEN");
             if (adminTokenStr == null || adminTokenStr.isEmpty())
                 adminTokenStr = System.getProperty("KB_ADMIN_AUTH_TOKEN");  // For tests
-            if (adminTokenStr != null && !adminTokenStr.isEmpty()) {
+
+            String miniKB = System.getenv("MINI_KB");
```

Comment:
Finally added support for mini-kb correctly

Reply:
This is required because mini-kb is currently configured to use the CI-Catalog, and the local admin token will not work with the CI catalog to get the volume mounts, as an admin token is required.